src/iop/demosaic.c

```diff
@@ -101,6 +102,8 @@ typedef struct dt_iop_demosaic_global_data_t
   int kernel_zoom_third_size;
   int kernel_vng_green_equilibrate;
   int kernel_vng_interpolate;
+
+  double CAM_to_RGB[3][4];
```

Comment:
I'm not sure we can/should use global data for that, it it *global*, so there are likely to be some 
I would recommend storing it in dt_iop_demosaic_data_t in commit_params()